commands/deploy.go

```diff
@@ -2,19 +2,91 @@ package commands
 
 import (
 	"fmt"
+	"io/ioutil"
+	"net/http"
 	"os"
+	"path/filepath"
+	"strings"
+	"time"
 )
 
-// DeployCmd handles deploying an app to Section
-type DeployCmd struct{}
+// MaxFileSize is the tarball file size allowed to be uploaded in bytes.
+const MaxFileSize = 1073741824 // 1GB
+
+// DeployCmd handles deploying an app to Section.
+type DeployCmd struct {
+	Debug     bool
+	ServerURL string `default:"https://api.section.io/code-upload/v1"`
+}
 
 // Run deploys an app to Section's edge
 func (a *DeployCmd) Run() (err error) {
-	path, err := os.Getwd()
+	fmt.Println("Server URL", a.ServerURL)
+	// path, err := os.Getwd()
```

Comment:
Let's just remove these lines. 

If we need them again, revert the commit, and submit a new PR.

Reply:
done